CHANGELOG.md

```diff
@@ -1,5 +1,64 @@
 # CHANGELOG
 
+## 2.0.0 (2022-11-14)
+
+MLflow 2.0.0 includes several breaking changes from the 1.x API, additional major features and improvements
+
+Breaking Changes:
```

Comment:
Can we call out breaking changes below features, since features are supposed to be the most exciting part of the release?

Reply:
Moved